src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy

```diff
@@ -1,16 +1,20 @@
 package com.axis.system.jenkins.plugins.downstream.yabv.BuildFlowAction
 
+import com.axis.system.jenkins.plugins.downstream.cache.BuildCache
```

Comment:
Nit; seems unnecessary

Reply:
Why?

Reply:
It is never used in this file.

Reply:
God. I don't now how it is possible, but 3 times I opened the wrong file and double checked that it was actually used. Sorry about that.